oap-server/analyzer/log-analyzer/src/main/java/org/apache/skywalking/oap/log/analyzer/dsl/spec/extractor/ExtractorSpec.java

```diff
@@ -290,13 +294,34 @@ public void slowSql(@DelegatesTo(SlowSqlSpec.class) final Closure<?> cl) {
             return;
         }
 
+        LocalDateTime localDateTime = Instant.ofEpochSecond(log.getTimestamp()).atZone(ZoneId.systemDefault()).toLocalDateTime();
+        String timeBucketForDB = DTF.format(localDateTime);
+        builder.setTimeBucket(Long.parseLong(timeBucketForDB));
```

Comment:
Relate to https://github.com/apache/skywalking/pull/9988/files#r1026572253. The two ways seem very different, but for nearly the same thing?

Reply:
I forgot Timebucket class also support second unit. I will fix it later.